README.md

```diff
@@ -17,29 +20,37 @@ Once the project is cloned to disk you can import into Android Studio:
 * Navigate to the **arcgis-maps-sdk-kotlin-samples/** folder and click **OK**.
 
 ## Accessing Esri location services
+
 Accessing Esri location services, including basemaps, routing, and geocoding, requires authentication using either an API Key or an ArcGIS identity:
- 1. API key: A permanent key that gives your application access to Esri location services. Visit your [ArcGIS Developers Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.
- 
-The Android samples in this repository have been structured to use an API key, set once, which will run in all samples. Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory (`/Users/<user_name>/.gradle/gradle.properties`). The API_KEY property should contain quotes around the key itself: `API_KEY = "YOUR_API_KEY"`
 
-2. ArcGIS identity: An ArcGIS named user account that is a member of an organization in ArcGIS Online or ArcGIS Enterprise.
+* API key
+
+A permanent key that gives your application access to Esri location services. Visit your [ArcGIS Developers Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.
+The Android samples in this repository have been structured to use an API key, set once, which will run in all samples.
+Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory (`/Users/<user_name>/.gradle/gradle.properties`).
+The API_KEY property should contain quotes around the key itself: `API_KEY = "YOUR_API_KEY"`
+
+* ArcGIS identity
+
+An ArcGIS named user account that is a member of an organization in ArcGIS Online or ArcGIS Enterprise.
 
 ## Run a sample
-Once you have set up your developer environment you can run any sample from within Android Studio by selecting the app module from the **Edit Configurations** drop down and clicking the **Run** button from the toolbar. 
+
+Once you have set up your developer environment you can run any sample from within Android Studio by selecting the app module from the **Edit Configurations** drop down and clicking the **Run** button from the toolbar.
 
 ### Build/Run sample from Gradle
+
 You can execute all the build tasks using the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) command line tool. It's available as a batch file for Windows (gradlew.bat) and a shell script for Linux/Mac (gradlew.sh) and it is accessible from the root of the project.  
 
-- Build a debug APK
+* Build a debug APK
 
-```
-$ ./gradlew assembleDebug
+```gradle
+./gradlew assembleDebug
```

Comment:
The code block is not `.gradle` code but just a terminal execution.

Reply:
this was to resolve a warning from code inspection. to have the language specified. My intention of adding `gradle` was to specify that its a `gradle command` for the terminal. Correct, its not a language, but then I dont know what else suits best here so we can address the warning.

Reply:
- Are you referring to "Inspect code"? Don't think we should modify non-code files using inspect code in the first place. 
- If you are referring to the GitHub [README_style_checker](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/tools/CI/README_Metadata_StyleCheck/README_style_checker.py), the warnings are designed to format the readme file for a new sample, not the top-level repository readme. 

In either case, if the code block has a language that can use [style highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) it should be used. This code block does not contain gradle code and probably needs to be removed.

Reply:
I was referring to GitHub [README_style_checker](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/tools/CI/README_Metadata_StyleCheck/README_style_checker.py), sorry for the typo.

Reply:
I reverted it based off your comment. 
But I don't get why we shouldn't resolve the warnings by the Github style checker for the top level Readme (vs samples Readme) as it's still part of the samples repo. I think it would be good if we are able to adhere to the Readme guidelines in general for the repo :)

Reply:
Sure, resolving obvious warnings is a no-brainer! And the heading changes in the PR are welcomed. To me, the expected highlighting style isn't that obvious from the [list](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) of accepted keywords. If possible resolving them is ideal, but the code block does serve its function as it is.